src/python/test/test_remote_datasets.py

```diff
@@ -49,6 +49,7 @@ def _capture_license_output(fetch_method, **kwargs):
     # Force file_path in forwarded arguments
     kwargs["file_path"] = "./tmp_for_test/data.npy"
     data_array = fetch_method(**kwargs)
+    del data_array
```

Comment:
Just checking: that's only necessary in this test because we remove the file, but it does not affect a regular user (unless maybe they use clear_data_home, but that doesn't seem too bad)? Looks ok.